ext/pdo_mysql/tests/pdo_mysql_stmt_blobfromstream.phpt

```diff
@@ -53,11 +53,11 @@ unlink($file);
                 return false;
             }
 
-            $db->exec('DROP TABLE IF EXISTS test');
-            $sql = sprintf('CREATE TABLE test(id INT, label BLOB) ENGINE=%s', PDO_MYSQL_TEST_ENGINE);
+            $db->exec('DROP TABLE IF EXISTS test_stmt_blobfromstream');
```

Comment:
This should be moved between the 2 calls to this function, or maybe even split into a new test IDK.

Also the set-up class seems to be common to some tests?